src/workshops/node-npm-intro/index.md

```diff
@@ -28,13 +28,17 @@ You'll need to have Node installed on your computer to follow this workshop. Che
 node --version
 ```
 
-You should see a version number printed. Ideally you want at least the current Long-term Support (LTS) version listed on [Node's homepage](https://nodejs.org/en/).
+You should see a version number printed. If you get an error that means Node isn't installed. Follow [our installation guide](/course/handbook/installation/#node) then try again.
 
-If you get an error that means Node isn't installed. Follow [our installation guide](/course/handbook/installation/#node) then try again.
+Our programme relies on some features that were only added in Node version 18 (the current version), so if you have an older version than that you should install the newer one with:
```

Comment:
Add this to the installation guide?

Reply:
Yeah good point, will update that in another PR